src/algorithms/time_evolution/evoltools.jl

```diff
@@ -1,28 +1,28 @@
 """
-    get_gate(dt::Float64, H::LocalOperator)
+    get_expham(dt::Float64, H::LocalOperator)
 
-Compute `exp(-dt * H)` from the nearest neighbor Hamiltonian `H`.
+Compute `exp(-dt * H)` from Hamiltonian `H`.
+Each term in `H` must be a single `TensorMap`.
 """
-function get_gate(dt::Float64, H::LocalOperator)
-    @assert all([
-        numin(op) == 2 && norm(Tuple(terms[2] - terms[1])) == 1.0 for (terms, op) in H.terms
-    ]) "Only nearest-neighbour terms allowed"
-    return LocalOperator(
-        physicalspace(H), Tuple(sites => exp(-dt * op) for (sites, op) in H.terms)...
-    )
+function get_expham(dt::Float64, H::LocalOperator)
+    return LocalOperator(H.lattice, (sites => exp(-dt * op) for (sites, op) in H.terms)...)
 end
 
 """
-    is_equivalent(
-        bond1::NTuple{2,CartesianIndex{2}},
-        bond2::NTuple{2,CartesianIndex{2}},
-        (Nrow, Ncol)::NTuple{2,Int},
-    )
+Check if a Hamiltonian contains only nearest neighbor terms
+"""
+function is_nnham(H::LocalOperator)
```

Comment:
```suggestion
    is_nearest_neighbour(H)

Check if an operator contains only nearest neighbor terms
"""
function is_nearest_neighbour(H::LocalOperator)
```

In general I think using longer names might be easier to read here, but I don't feel that strongly about it.
Adding the signature in the docstring is a good idea though

Reply:
Are we following the British spelling in the repo? (`neighbour` instead of `neighbor`) But I also see that `center` instead of `centre` is used...

Reply:
I think British and American spelling are mixed throughout the repo, so at this point you can use what you prefer. (Perhaps we can unify at some point but I dont think that's a super high-priority issue right now.)